client/v2/go.mod

```diff
@@ -156,3 +156,5 @@ require (
 )
 
 replace github.com/cosmos/cosmos-sdk => ./../../
+
+replace cosmossdk.io/x/gov => ./../../x/gov
```

Comment:
If you remove the types/mempool dep, we wouldn't need that.

Reply:
addressed, thanks for the review